temporal-sdk/src/main/java/io/temporal/internal/client/WorkflowClientLongPollHelper.java

```diff
@@ -46,10 +45,11 @@
 import java.util.concurrent.TimeoutException;
 
 /** This class encapsulates sync long poll logic of {@link RootWorkflowClientInvoker} */
-class WorkflowClientLongPollHelper {
+final class WorkflowClientLongPollHelper {
   /**
    * Returns result of a workflow instance execution or throws an exception if workflow did not
-   * complete successfully.
+   * complete successfully. Will wait for continued-as-new generations of the original workflow
```

Comment:
```suggestion
   * complete successfully. Will wait for continued-as-new instances of the original workflow
```

Reply:
Btw might be worth knocking out https://github.com/temporalio/sdk-java/issues/696 in here

Reply:
We call them executions. Each execution consists form 1 or many runs.